src/scripts/postgresql.conf

```diff
@@ -0,0 +1,799 @@
+# -----------------------------
```

Comment:
once you rebase with the `pg_cron` change we will need to add in this file 2 lines:
```
shared_preload_libraries=pg_cron
cron.database_name=iasql_metadata
```

Reply:
Done :)